x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -750,10 +764,51 @@ export class TelemetryReceiver implements ITelemetryReceiver {
             },
           },
         },
-      },
-    };
+        track_total_hits: false,
+        sort: [
+          { '@timestamp': { order: 'asc', format: 'strict_date_optional_time_nanos' } },
+          { _shard_doc: 'desc' },
+        ] as unknown as string[],
+        pit: { id: pitId },
+        search_after: searchAfter,
+        size: 1000,
+      };
+
+      this.logger.debug(`Getting alerts with point in time (PIT) query: ${JSON.stringify(query)}`);
```

Comment:
we can use tlog for any logs here so we log as info level for cloud customers; hence, collecting them more readily into our monitoring cluster

Reply:
Thanks! See commit [36769ce](https://github.com/elastic/kibana/pull/152998/commits/36769ceedaf1fd6732d2e83e3005b4fa6925575c)